custom_metrics/a11y.js

```diff
@@ -210,4 +210,184 @@ return JSON.stringify({
   screen_reader_classes: captureAndLogError(() => {
     return document.querySelectorAll('.sr-only, .visually-hidden').length > 0;
   }),
+  form_control_a11y_tree: captureAndLogError(() => {
+    function doesMatchAny(string, regexps) {
+      let found = false;
+      for (const regexp of regexps) {
+        if (regexp.test(string)) {
+          return true;
+        }
+      }
+
+      return false;
+    }
+
+    const attributes_to_track = [
```

Comment:
Might be simpler to bundle all of these in a single regex that can be tested at once, rather than requiring iteration. For example: `/^(aria-.+|type|id|...)$/i`. Then `doesMatchAny` can be replaced with `attributes_to_track.test(key)`. I've also added the `i` flag to handle case insensitivity rather than lowercasing the key manually.

Reply:
Good idea. Made this change in the latest commit